src/main/java/frc/robot/RobotContainer.java

```diff
@@ -45,8 +46,9 @@ public RobotContainer() {
     DoubleSupplier rot = () -> Utils
         .oddSquare(Utils.deadZone(-m_driveController.getRightX(), OIConstants.kJoystickDeadzone))
         * SwerveConstants.kMaxAngularSpeedRadiansPerSecond * 0.2;
-    m_swerveDriveSubsystem.setDefaultCommand(
-        new SwerveDriveCommand(m_swerveDriveSubsystem, x, y, rot, () -> m_driveController.getRightBumper()));
+
+    m_swerveDriveSubsystem.setDefaultCommand(new RunCommand(() -> m_swerveDriveSubsystem.drive(x.getAsDouble(),
```

Comment:
Why does passing in a double supplier work?  Did you change the drive method to take in double suppliers instead of doubles?

Reply:
It's not passing in a Double Supplier, it's passing in DoubleSupplier.getasDouble